### ISSUE TITLE
Save location not being used in all situations in Engineering Diffraction UI

### ISSUE THREAD

**Issue by DannyHindson** [mantidproject/mantid#33877]
a) clear down all EngineeringDiffraction2 settings in MantidWorkbench.ini fle
b) start up workbench
c) open up Engineering Diffraction UI
d) Create New Calibration on Calibration tab using run 305738

Some .nxs and .prm files should be saved into C:\\Users\\iao46725\\Engineering_Mantid\Calibration\ as indicated by the "Save Location" setting within the settings for this UI (cog button on bottom left of engineering diffraction UI)

Although the default save location has been set up as something like C:\Users\iao46725\Engineering_Mantid\ it doesn't get used to save the files. Some of the files are instead saved into a Calibration subdirectory of whatever the current working directory happens to be - this can vary. And some don't get written at all. The following error is also produced:

![image](https://user-images.githubusercontent.com/56295817/167885689-da427ae5-d10f-4ca2-966a-150eed401ec9.png)

If you shut down workbench and start it up again it works OK.

Initial investigation suggests the result of output_settings.get_output_path() is a blank string on first start up of workbench (called in qt/python/mantidqtinterfaces/mantidqtinterfaces/Engineering/gui/engineering_diffraction/tabs/calibration/model.py). The default doesn't seem to kick in properly at first.

**Comment by DanielMurphy22** [comment]
Can reproduce on Ubuntu 18.04 using:

```
cd $HOME/.config/mantidproject
mv mantidproject.ini mantidproject.ini.backup
cd $HOME
mv .mantid .mantidbackup
```

**Comment by DanielMurphy22** [comment]
Fixed by #33928
